openid-4-verifiable-presentations-1_0.md

```diff
@@ -792,6 +794,10 @@ more information.
 If the Wallet cannot deliver all claims requested by the Verifier
 according to these rules, it MUST NOT return the respective Credential.
 
+Note: For Credential Formats that do not support selective disclosure, the case of both `claims`
+and `claim_sets` being absent is interpreted as requesting a presentation of the "full credential"
+since all claims are mandatory to present.
```

Comment:
```suggestion
For Credential Formats that do not support selective disclosure, the case of both `claims`
and `claim_sets` being absent is interpreted as requesting a presentation of the "full credential"
since all claims are mandatory to present.
```